internal/template.go

```diff
@@ -51,13 +49,8 @@ func ExecuteTemplate(sourceFilePath string, funcMap template.FuncMap, verbose bo
 
 // GetFuncMap builds the relevant function map to helm_ssm
 func GetFuncMap(profile string) template.FuncMap {
-	e := engine.New()
-
 	// Clone the func map because we are adding context-specific functions.
 	var funcMap template.FuncMap = map[string]interface{}{}
-	for k, v := range e.FuncMap {
```

Comment:
FuncMap is (by design) private in helm3. cf. https://github.com/helm/helm/issues/5779